cluster-service/update_instance_types.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python3
+
+import os
+import json
+import yaml
+from yaml.resolver import BaseResolver
+
+class AsLiteral(str):
+  pass
+
+def represent_literal(dumper, data):
+  return dumper.represent_scalar(BaseResolver.DEFAULT_SCALAR_TAG, data, style="|")
+
+yaml.add_representer(AsLiteral, represent_literal)
+
+region = "westus3"
```

Comment:
Might be useful to make this configurable so that it can be used in other regions as well

Reply:
added argparse to make those things configurable, so you guys can generate it per-region in the future, also with different output paths.